File: lol/src/main/kotlin/com/your/lol/dto/ParticipantDto.java
package com.your.lol.dto;

import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class ParticipantDto {
    private long physicalDamageDealt;
    private long physicalDamageTaken;
    private long champExperience;
    private double kda;
    private String summonerId;
    private long summonerLevel;
    private String summonerName;
    private ChallengesDto challenges;
}

File: ggtics-recruitement-lol-score/src/main/kotlin/com/your/lol/dto/riot/ParticipantDto.kt
package com.your.lol.dto.riot

class ParticipantDto(
        private val summonerName: String,
        val teamId: Int,
        val championName: String,
        val gameDuration: Long,
        val challenges: ChallengesDto?) {

    private constructor() : this("", 0, "", 0, null)

    fun equalsId(summonerName: String): Boolean {
        return this.summonerName == summonerName
    }
}